app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -446,7 +446,7 @@
 
   <div class="field">
     <label>{{t 'Event Type'}}</label>
-    {{#ui-dropdown class='search selection' selected=data.event.type onChange=(action (mut data.event.type)) as |execute mapper|}}
+    {{#ui-dropdown class='search selection' allowAdditions=true selected=data.event.type.name onChange=(action (mut data.event.type)) as |execute mapper|}}
```

Comment:
We do not allow additionsFor this. The user must not be able to add their own type.